tests/test_resolutions.py

```diff
@@ -33,6 +33,10 @@ def _assert_descriptions(obj: BaseResolution, element: WebElement,
     assert describe_none.out == expected.describe_none
 
 
+def assert_matcher_annotation(obj: BaseResolution):
+    assert type(obj.matcher) is obj.__annotations__['matcher']
```

Comment:
This is making me think it might be useful to have a concept of testing tools for ScreenPy plugins, since you had to make the same function in your ScreenPy PR.

Oh, wait, there's this: https://github.com/ScreenPyHQ/screenpy/blob/trunk/screenpy/test_utils.py

Maybe we could put this in there, and update the dependency? What do you think about that approach?